### ISSUE TITLE
Signing transactions with Keplr error

### ISSUE THREAD

**Issue by fadeev** [tendermint/vue#128]
> I'm facing a weird issue, where I'm able to send transactions via the VUE app, but as soon as I try using the Keplr is fails with a error 501.

> The front end shows:

```
Could not broadcast Tx: Transaction with ID 69FF1824CD271879CB9D3E2901CFB40DB1FF9AA115D1CFB15DEAF62C8073DDCE was submitted but was not yet found on the chain. You might want to check later
```

![unknown](https://user-images.githubusercontent.com/332151/140699893-f76a7cb4-8054-4250-916f-68a6f980761e.png)

> There is a full node connected to the validators, getting data from them... The urls on the VUE app are:

```
VUE_APP_API_COSMOS=https://rest-ufo.alien.blue
VUE_APP_WS_TENDERMINT=wss://wsrpc-ufo.alien.blue/websocket
VUE_APP_API_TENDERMINT=https://rpc-ufo.alien.blue
VUE_APP_ADDRESS_PREFIX=alien
VUE_APP_CHAIN_ID=moon
```

> I think Kepler is expecting different endpoints maybe? does it really matter?   It's def an issue with Kepler but I'm not sure if you or anyone else has encountered this issue.

cc @daniel-farina

**Comment by daniel-farina** [comment]
I'm adding more details here to reproduce the issue. I'm adding some seed phrases for the team to test it. This is fine as this chain will get deleted. 

This is an out of the box vue app from starport. 

The .env for the VUE app is: 

```
VUE_APP_API_COSMOS=https://rest-ufo.alien.blue
VUE_APP_WS_TENDERMINT=wss://wsrpc-ufo.alien.blue/websocket
VUE_APP_API_TENDERMINT=https://rpc-ufo.alien.blue
VUE_APP_ADDRESS_PREFIX=alien
VUE_APP_CHAIN_ID=alien
VUE_APP_CHAIN_NAME="Alien"

```

## What works
Sending from bash works without any issues:

Chain-id: alien

```
aliend tx bank send alien1ajay55sd4474vrs97fesqky9wt0rf9epyfw4ce alien1dut3t6c4m3m4zck7nr8mdm6an56xls26rw5yft 1000alien --chain-id alien --keyring-backend test
```
Also sending from the Vuejs's native wallet works perfectly. 

## The error - Steps to reproduce: 

### Background
I deleted the local store, and create a brand new Keplr wallet, new browser session, etc.

Visit: https://dapp.alien.blue/

Load this wallet to Keplr: 
```
awesome problem genre agree paddle field banana witness wink business ask term hair doctor tree pond churn burden series involve swift author stem today

```
Now you are a millionare, jk. 

Then connect the wallet like always
![image](https://user-images.githubusercontent.com/13665117/140705889-078be85d-76c6-43cd-b123-5b095062dfdc.png)

Then send 1000 to this address

```
alien1dut3t6c4m3m4zck7nr8mdm6an56xls26rw5yft
```

![image](https://user-images.githubusercontent.com/13665117/140706111-1b199ce6-23d8-4fcb-8f30-13bda4a68d32.png)

This what the payload looks like:

```
{
  "txBody": {
    "messages": [
      {
        "fromAddress": "alien1ajay55sd4474vrs97fesqky9wt0rf9epyfw4ce",
        "toAddress": "alien1dut3t6c4m3m4zck7nr8mdm6an56xls26rw5yft",
        "amount": [
          {
            "denom": "alien",
            "amount": "10000"
          }
        ]
      }
    ],
    "memo": ""
  },
  "authInfo": {
    "signerInfos": [
      {
        "publicKey": {
          "type_url": "/cosmos.crypto.secp256k1.PubKey",
          "value": "CiEDMUDoTCc+uaWzRAOi37p3v3/A4zNshFlbBlIES5qwvNs="
        },
        "modeInfo": {
          "single": {
            "mode": "SIGN_MODE_DIRECT"
          }
        },
        "sequence": "7"
      }
    ],
    "fee": {
      "amount": [
        {
          "denom": "alien",
          "amount": "5000"
        }
      ],
      "gasLimit": "200000"
    }
  },
  "chainId": "alien",
  "accountNumber": "0"
}

```

It will sit there processing for a while.....
![image](https://user-images.githubusercontent.com/13665117/140706233-524c5abf-0e34-4cbe-b1a1-38107fcc0132.png)

Then this error shows on the console:

![image](https://user-images.githubusercontent.com/13665117/140706348-133a4cb2-ecee-4e8f-911a-5e0cc27dad27.png)


Also, if you send a transaction  from Kepler: 

![image](https://user-images.githubusercontent.com/13665117/140706483-edd77309-9429-499e-9540-96511b9909a2.png)

It shows a error as well:

![image](https://user-images.githubusercontent.com/13665117/140706537-ba0f676f-8415-4a6e-a0aa-4b86254cdeb1.png)

**Comment by marinhoarthur** [comment]
We'll be experimenting with `no-legacy-stdTx` in [here](https://github.com/tendermint/vue/blob/66025e2ddb5e4c39c8d5f5f639104c25cab1a906/packages/vue/src/components/SpWalletCreate/SpWalletCreate.vue#L306)

As soon as we have updates on this, will be posted here.

cc @daniel-farina

**Comment by daniel-farina** [comment]
Great,
I was just testing that last night based on [167](https://github.com/chainapsis/keplr-extension/issues/167). 

You should be able to use the .env above and run the vue locally against the testing chain we made. 

Thank you

**Comment by marinhoarthur** [comment]
Adding the `no-legacy-stdTx` flag works. We'll prepare a release of the frontend. Soon enough it's deployed and I'll update this issue.

**Comment by daniel-farina** [comment]
Great news! We will test it on our side.

**Comment by marinhoarthur** [comment]
Released under [v0.1.55](https://github.com/tendermint/vue/releases/tag/v0.1.55)

**Updating your `@starport/vue` package is necessary to get the fix.**

cc @daniel-farina

**Comment by daniel-farina** [comment]
awesome job @marinhoarthur 
Thank you for working with us and keeping us posted along the way.
